include/dlaf/matrix/panel.h

```diff
@@ -262,6 +272,18 @@ struct Panel<axis, const T, D> {
     dim_ = height;
   }
 
+  /// Get the current width of the col panel.
+  template <Coord A = axis, std::enable_if_t<A == axis && Coord::Col == axis, int> = 0>
+  SizeType getWidth() const noexcept {
```

Comment:
Is this again `cols`? I'm not necessarily asking you to change, but just checking. This might be something that will be difficult to have consistent... I suppose the reason for this naming is that the setters are `setHeight/Width`.

Reply:
Exactly, to be consistent with the setters.

The setter were called setWidth/setHeight because they have a specific behaviour and they don't match any size.cols()/rows() you can get from the panel (or its parent distribution).

Anyway, in my plans for the API refactoring these methods will disappear, as I want to unify Vector, Panel and Matrix.